src/plugins/data/common/field_formats/converters/string.ts

```diff
@@ -6,7 +6,7 @@
  * Side Public License, v 1.
  */
 
-import escape from 'lodash/escape';
```

Comment:
Fixing wrong import. Should be less now, because 'lodash' included in shared deps